Mathlib/Order/Basic.lean

```diff
@@ -66,7 +65,7 @@ open Function
 
 universe u v w
 
-variable {ι : Type*} {α : Type u} {β : Type v} {γ : Type w} {π : ι → Type*} {r : α → α → Prop}
+variable {ι : Type*} {α : Type u} {β : Type v} {π : ι → Type*}
```

Comment:
While you're at it
```suggestion
variable {ι α β : Type*} {π : ι → Type*}
```

Reply:
Do you want to get rid of `universe u v w` altogether?

Reply:
I deleted them.